lib/statsd/instrument/metric.rb

```diff
@@ -103,6 +103,6 @@ def inspect
   def self.normalize_tags(tags)
     return unless tags
     tags = tags.map { |k, v| k.to_s + ":".freeze + v.to_s } if tags.is_a?(Hash)
-    tags.map { |tag| tag.tr('|,'.freeze, ''.freeze) }
+    tags.map { |tag| tag.to_s.tr('|,'.freeze, ''.freeze) }
```

Comment:
Why is this necessary? This is a very hot code path for us, and we'd like to not add any unnecessary processing here.